app/src/main/java/org/mozilla/focus/ext/Context.kt

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.ext
+
+import android.content.Context
+
+// Extension functions for the Context class
+
+val Context.appVersionName: String?
+    get() {
+        val packageInfo = packageManager.getPackageInfo(packageName, 0)
+        return packageInfo.versionName
+    }
+
+fun Context.dp(px: Int): Int = (px * resources.displayMetrics.density).toInt()
```

Comment:
The name `dp` is confusing unless you use a argument label on the callsite (e.g. `context.dp(px = 8)`). However, since you can't force argument labels in Kotlin, I'd prefer if the name was more descriptive, `Context.pxToDP`.